server/main.go

```diff
@@ -16,9 +16,10 @@ import (
 
 // Command line arguments (global variables)
 var (
-	enroll   = flag.Bool("enroll", false, "Must be true for a first time attestation")
-	loglevel = flag.Int("loglevel", 0, "Indicates the level of logging, 0 is the minimum, 3 is the maximum")
-	mtu      = flag.Int("mtu", 500, "Set a custom MTU, which is basically the max size of the BLE packets")
+	enroll       = flag.Bool("enroll", false, "Must be true for a first time attestation")
+	loglevel     = flag.Int("loglevel", 1, "Indicates the level of logging, 0 is the minimum, 3 is the maximum")
```

Comment:
I changed the loglevel to 1 by default so that the user is aware of the attestation state. If using ultrablue for disk decryption, it provides visual information so that the user knows what is going on.